src/styles.css

```diff
@@ -1,6 +1,43 @@
-.hello {
-    color: red;
-  }
-  body {
-    background-color: bisque;
-}
\ No newline at end of file
+.container {
+  width: 30%;
+  background-color: #fff;
+  margin: 10% auto;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+body {
+  background: rgb(198, 195, 195);
+}
+
+table {
+  border-collapse: collapse;
+}
+
+tbody tr {
+  border-bottom: 1.8px solid rgb(174, 167, 167);
+  height: 35px;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+}
```

Comment:
> - [ ] The alignment of the table row descriptions appears to be centered. It would be great if you could ensure that the alignment matches the provided design. This is important because maintaining consistency with the design creates a cohesive and professional user interface, enhancing the overall user experience.😊
> 
> Given Design:
> 
> ![b](https://github.com/fatmahussein/To-do-list/assets/40009354/38948066-6ab6-41a4-8aef-11cb742bf8d0)
> 
> Your design:
> 
> ![br](https://github.com/fatmahussein/To-do-list/assets/40009354/6ed3bb95-2b68-4d11-9c71-736625ed6e3f)
>